.github/mergify.yml

```diff
@@ -0,0 +1,12 @@
+pull_request_rules:
+  - name: automatic merge for dependabot pull requests
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
+      - "status-success=unit-tests"
+      - "status-success=CodeQL Analyze (javascript)"
+      - "status-success=DCO"
+      - "status-success=WIP"
```

Comment:
Listing all merge conditions like this is kind of unfortunate. If we add another required check in the future, we'd need to remember to add it here too.

Does it not support "merge on green" where "green" is defined by the required checks on the repo, whatever they are?

Reply:
It might now, I think we could determine the conditions on the repo settings and mergify just won't be able to merge it. We could give it a try afterward.

Reply:
>Note that Mergify will always respect the branch protection settings. When the conditions match and the merge action runs, Mergify waits for the branch protection to be validated before merging the pull request.

It might just work, I'll remove this from this PR.